q2_quality_control/_blast.py

```diff
@@ -66,14 +64,14 @@ def _extract_hits(blast_output):
     '''
     with open(blast_output, "r") as inputfile:
         # grab query IDs from each line (only queries with hits are listed)
-        hits = [line.split('\t')[0] for line in inputfile
+        hits = {line.split('\t')[0] for line in inputfile
                 # ignore comment lines and blank lines
                 if not line.startswith('#')
                 and line != ""
                 # if vsearch fails to find assignment, it reports '*' as the
                 # accession ID, so we will not count those IDs as hits.
-                and line.split('\t')[1] != '*']
-    return hits
+                and line.split('\t')[1] != '*'}
+    return list(hits)
```

Comment:
Why list instead of `set`? Note that if you're doing lookups on this (e.g., `if x in hits`), a `set` will be orders of magnitude faster than a list.

Reply:
this is now a set in #11  — just a bit of merge conflict that I will take care of when the time comes